package/koreader/package

```diff
@@ -18,12 +18,14 @@ source=(
     koreader.draft
     KOReader.oxide
     rm2-support.patch
+    koreader
```

Comment:
what is this doing?

Reply:
`$recipedir` is not supported by the new implementation, since it was just an implementation detail of the Bash tooling. This change makes the `koreader` recipe use the correct mechanism for importing sources from the recipe directory.